.github/workflows/trunk-based-flow.yml

```
@@ -0,0 +1,272 @@
+env:
+  NODE_VERSION: '20.x'
+  TURBO_TOKEN: ${{ secrets.TURBO_TOKEN }}
+  TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
+
+jobs:
+  # Job 1: Quality Gates (runs on all PRs)
+  quality-gates:
+    name: Quality Gates
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.draft == false
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)